styles.css

```
@@ -0,0 +1,927 @@
+  z-index: 20;
+
+  pointer-events: none;
+}
+
+/* =========================
+   SMOOTH COUNTDOWN
+========================= */
+
+.countdown {
+
+  position: absolute;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)